set.mm

```diff
@@ -515900,6 +516454,485 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+        YTVGWWHWWMWYQXUJXULXUFYTWWHVYCWYQXUJXULWWJYTXUMXHXUMWYTVVJWVAXUMWYTVVKV
+        VJWYRXUJXULXEZXUMVVKVVJVWTWWHWYQXUJXULYOZXCXTXUMWVBUGUDVBZYSZWYTVXEUSYS
+        ZWYTWVAUSYSZWWGWVBWYQXUJXULYMXUMWYTVVKVWTXUNXUMVVKVVJVWTXUOXDUWKXUMWYTV
+        XDUWLXURXUMWYTVXDXUKXULVFUWMWYTVXDUWNUQWVBXUSXUQXURVAWVAVWTVXEWYTUXQUWO
+        UWPXFXIUWQWYRVVKVVJVWTWWHWYQVFXCZWYRXHUWRUWSVNWYRVYRVYSWWHWYQWUIVYRXUTW
+        XEYNWUHUQXMWXMWXBDGEWYKVVNMOWXCWXBXUDWYKWHWXBVVKXUCWYJWWHXUEWUIXUHXBYRW
+        WHVVJWHVVKXUCWWHUGVVJXUBWHWWHWYTVVJUSZVAZWYSVGXUAWHXVBWYSVAXHXVBXULVAVV
+        JWHWYTWUTWWHWWNXVAXULWXFXAWWHXVAXULXEXRUWTUXAUXBUXCWXEXOWXSXPYLWVOWYOWV
+        PEVDWVKWVMWVBWYOWVPUMVXCWWHBVWTWYLWVDWXOWYKWVCVVNGWXOXUDWYKWVCWXOVVKXUC
+        WYJWWHXUEWXNXUHXBYRWXOUGVVKXUBWVCVVJXUCWHXUIWXOXUJVAZXUBXUAWVCXVCWYSVGX
+        UAXVCXUPUEUDVBYSZXULXVCWYTVVKVWTWXOXUJVFZWWHWXNXUJXEXTWWHXVDWXNXUJWYCXA
+        WYTVXDVWTUXDWSUXEXVCWYTVVKWUTXVEUXFXMWXTWYAUWRUWSVNUXGYLVPXMWVOWYJVGVHV
+        MZVWTUMZWYNVUPUSZWVKWVMWVBXVGVXCWWHXVFWYGVGVHVMZWYIVGVHVMZWAVWTVSWAZVWT
+        WWHWYGWYIWFWFWHVGWWHWUTWYFJWVKWVMWVBXEUXHWYIWFUSWWHWYHUXIUNXUGUXJWWHXVI
+        VWTXVJVSWWHXVIWVAVXEWRVXFVXEWRZVWTWWHVVJVXEWUTWFWHVGWWJXUGXUFUXKWWHWVAV
+        XFVXEWWGWVBVFUXLWWHXVDXVLVWTUMZWYCXVDVWTVXEUXMVSUMXVMVWTVXDUXNVWTVXEUXO
+        UXPUQXQXVJVSUMWWHVXEVGVTUXRZVGVHVMXVJVSXVNWYIVGVHVXDVGUEUXSUXTUYAUYBVXE
+        VGUYCUYDUNUYEXVKVWTUMWWHVWTUYHUNXQYLZWVOVXBXVGXVHURHJWYJVWEWYJUMZVXAXVG
+        VWLXVHXVPVWFXVFVWTVWEWYJVGVHVJVKXVPVWKWYNVUPXVPVWJWYMEVDXVPBVVJVWIWYLXV
+        PVWHWYKVVNGVVKVWEWYJVLVNVOVPVQVRWVKVXCWVMWVBYOWVOWYJVYLHVYIUKJWVOVYLWYJ
+        VGWKWLHWYJVYIVWEWYJVGWKYCWVOWHVVJWYJWFWFVYMWVOWOUNWVKWVMWVBVYCVXCWWJYLW
+        VOVVJWHXUCWYJWVKWVMWVBXUEVXCXUHYLWVOUGVVJXUBWHWVOXVAVAZWYSVGXUAWHXVQWYS
+        VAXHXVQXULVAZVVJWHWYTWUTXVRAWVMWWNAVVCVUQVVSWUQWUSVXCWVMWVBXVAXULUYFWVL
+        WVMWVBXVAXULYMWWRWSWVOXVAXULXEXRUWTUXAUYIZUYGZWVOWYJVYIWHVGXVTWVOXHWVOV
+        VJWHWYJXVSUYQWVOXVFVWTXNXVOWVKWVMWVBVWTXNUSVXCWXMYLXTUYJUPQUYKUYLUYMUXC
+        WVOWVSDEVUPGWVRMOAWUKVVCVUQVVSWUQWUSVXCWVMWVBWULYFWVOVUPKUYNUHUSZVXDVUP
+        VCZUSWVSVUPUSVVCXWAAVUQVVSWUQWUSVXCWVMWVBVUPEKNUYOYQWVOVVJXWBVXDVUQVVJX
+        WBUJVVDVVSWUQWUSVXCWVMWVBCVUPXJUYRWVKWVMWVBVXDVVJUSVXCWYDYLZXKVUPKVXDUY
+        PWSWYEWVOVVJWHVXDWUTWVKWVMWWNVXCWVBWWSUYSXWCXRUYTVUEXTYPYAYPVUAWVHVXHUF
+        HJUFHVIZWVBVXGWVGVWLXWDWVAVWFVXFWUTVWEVGVHVJVKXWDWVFVWKVUPXWDWVEVWJEVDX
+        WDBVVJWVDVWIXWDWVCVWHVVNGVVKWUTVWEVLVNVOVPVQVRVUBVUCWXKVUDVVEVVSVFUYLUY
+        MUYSXTVVIIJVVQVVFIJVVQVEZXWEWDVVGVVFXWEVUFZUSZVVEVVGXWGFXWFVVFTVUGVUHWN
+        VUIVUJYPVUKYPYAVUQUAFVURVULVUMVUNXM $.
+    $}
+
+    $d .x. f i v $.  $d .x. g h w $.  $d A f i v $.  $d A g h w $.
+    $d B f i v $.  $d F f i v $.  $d F g h $.  $d M f i v $.  $d M g h w $.
+    $d R f i v $.  $d R g h $.  $d X g $.  $d f i ph v $.  $d h i v w $.
+    elrgspn.1 $e |- ( ph -> X e. B ) $.
+    $( Membership in the subring generated by the subset ` A ` .  (Contributed
```

Comment:
Would be nice to have a bit more explanations in the comment. If I understood the formalization correctly, maybe add something like

"An element ` X ` lies in the subring generated by a subset ` A ` if and only if ` X ` is a sum of products of elements from ` A `."

Reply:
Yes, almost. This description of the formula would miss for example (the group inverse) `- X`, which must be in the generated subring of `{ X }`. Wikipedia describes it as the linear combinations with integer coefficients of products of elements of `A`.